velox/vector/fuzzer/VectorFuzzer.h

```diff
@@ -215,7 +215,10 @@ class VectorFuzzer {
 
   // Returns a RowVector based on the provided vectors, fuzzing its top-level
   // null buffer.
-  RowVectorPtr fuzzRow(std::vector<VectorPtr>&& children, vector_size_t size);
+  RowVectorPtr fuzzRow(
```

Comment:
Would it make sense to provide a separate (overloaded) API so we don't break backwards compatibility (one with 2 and one with 3 parameters)?